pkg/discovery/grpcresolver/grpc_resolver.go

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) 2019 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://wwr.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package grpcresolver
+
+import (
+	"math/rand"
+	"strconv"
+	"sync"
+	"time"
+
+	"github.com/tysontate/rendezvous"
```

Comment:
do we need this dependency? I think I described the algorithm on the previous ticket, it just needs hash and sort from stdlib.

Reply:
yeah essentially the package is doing the same thing. Will replace it in next commit.